### PR TITLE
de.haeckerfelix.gradio

### PR DESCRIPTION
**Pull request by haecker-felix**

### CODE REVIEW

File: de.haeckerfelix.gradio.json [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://anongit.freedesktop.org/git/gstreamer/gst-devtools.git",
+                    "branch": "1.8"
+                }
+            ]
+        },
+        {
+            "name": "meson",
```
Comment (bil-elmoussaoui) [inline de.haeckerfelix.gradio.json]: Meson is already there so no need to add it

Comment (haecker-felix) [reply de.haeckerfelix.gradio.json]: Yeah you're right.

File: de.haeckerfelix.gradio.json [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        "--share=network",
+        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--device=dri",
+        "--talk-name=org.freedesktop.Tracker1",
+        "--socket=pulseaudio",
+        "--talk-name=org.gtk.vfs", "--talk-name=org.gtk.vfs.*",
+        "--talk-name=org.gnome.ScreenSaver",
+        "--talk-name=org.gnome.SettingsDaemon",
+        "--talk-name=org.mpris.MediaPlayer2",
```
Comment (TingPing) [inline de.haeckerfelix.gradio.json]: Why would it need to talk to this? It owning its own name below should be all for mpris.

File: de.haeckerfelix.gradio.json [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+    "build-options" : {
+        "cflags": "-O0 -g",
+        "cxxflags": "-O0 -g",
+        "env": {
+            "GST_PLUGIN_SYSTEM_PATH": "/app/lib/gstreamer-1.0/"
+        }
+    },
+    "modules": [
+        {
+            "name": "gstreamer",
```
Comment (TingPing) [inline de.haeckerfelix.gradio.json]: gstreamer is already in the runtime, is there anything you are missing from it?

File: de.haeckerfelix.gradio.json [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://anongit.freedesktop.org/git/gstreamer/gst-plugins-bad.git",
+                    "branch": "1.8"
+                }
+            ]
+        },
+        {
+            "name": "yasm",
```
Comment (TingPing) [inline de.haeckerfelix.gradio.json]: Already in the Sdk.

File: de.haeckerfelix.gradio.json [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+                {
+                    "type": "git",
+                    "url": "https://anongit.freedesktop.org/git/gstreamer/gst-devtools.git",
+                    "branch": "1.8"
+                }
+            ]
+        },
+      	{
+
+            "name": "gradio",
```
Comment (TingPing) [inline de.haeckerfelix.gradio.json]: Minor whitespace issues.

File: de.haeckerfelix.gradio.json [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+        "--talk-name=org.mpris.MediaPlayer2",
+        "--talk-name=org.mpris.MediaPlayer2.Player",
+        "--own-name=org.mpris.MediaPlayer2.gradio",
+        "--talk-name=org.gnome.SettingsDaemon",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.gnome.SessionManager",
+	"--filesystem=xdg-music:ro"
+    ],
+    "build-options" : {
+        "cflags": "-O0 -g",
```
Comment (TingPing) [inline de.haeckerfelix.gradio.json]: Don't disable optimizations.

File: de.haeckerfelix.gradio.json (hunk 3 above)
Comment (haecker-felix) [reply de.haeckerfelix.gradio.json]: Where i can see which codecs are included in the runtime? Or where can I generally see which software is included?

Comment (TingPing) [reply de.haeckerfelix.gradio.json]: You can look here: https://github.com/GNOME/gnome-sdk-images/tree/gnome-3-24

Or `flatpak run org.gnome.Sdk//3.24` and dig around the filesystem yourself.

**Comment by haecker-felix** [comment]
All issues should be fixed now.

File: org.ffmpeg.FFmpeg.json [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+      "--enable-gnutls",
+      "--enable-avfilter",
+      "--enable-avcodec",
+      "--enable-avformat",
+      "--enable-decoder=aac"
+   ],
+   "sources":[
+      {
+         "type":"archive",
+         "url":"https://ffmpeg.org/releases/ffmpeg-3.3.tar.xz",
```
Comment (TingPing) [inline org.ffmpeg.FFmpeg.json]: 3.3.3 is the current release.

**Comment by TingPing** [comment]
Test build: https://flathub.org/builds/#/builders/1/builds/359

**Comment by haecker-felix** [comment]
Everything "SUCCESS" 👍

**Comment by TingPing** [comment]
Repository has been created: https://github.com/flathub/de.haeckerfelix.gradio

**Comment by haecker-felix** [comment]
Thanks!

**Pull request closed without merging**